docs/deploying.md

```diff
@@ -65,11 +65,11 @@ Move and/or rename your crt file to: `certs/domain.crt` - and your key file to:
 
 Make sure you stopped your registry from the previous steps, then start your registry again with TLS enabled:
 
-	docker run -d -p 5000:5000 --restart=always --name registry \
-	  -v `pwd`/certs:/certs \
-	  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
-	  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
-	  registry:2
+  	docker run -d -p 5000:5000 --restart=always --name registry \
+  	  -v `pwd`/certs:/certs \
+  	  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
+  	  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
+  	  registry:2
```

Comment:
I'm accustomed to four spaces at the beginning of the line to create these code blocks in markdown.